tests/test_memory_leak.py

```diff
@@ -13,4 +13,5 @@ def test_rss_usage():
     for _ in range(1000):
         DirtyMixinModel()
     rss_2 = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss
-    assert rss_2 == rss_1, 'There is a memory leak!'
+    diff_percentage = 100 * abs(rss_2 - rss_1) / rss_1
+    assert diff_percentage <= 1, 'There is a memory leak!'
```

Comment:
Why this change?

Reply:
I'm not entirely convinced that an exact match is the only indication of no memory leak, because of garbage collection. Specifically because it was starting to fail all the time with minuscule amounts.